components/edit-collective/sections/Tiers.js

```diff
@@ -165,6 +167,42 @@ const Tiers = ({ collective }) => {
                 )}
               </Mutation>
             </Box>
+            {cryptoContributionsEnabledByHost && (
+              <Box mb={4}>
+                <StyledHr my={4} borderColor="black.300" />
+                <P fontSize="14px" lineHeight="20x" mb={3}>
+                  <FormattedMessage defaultMessage="Enabling this will add a tier that allows users to support your collective with Cryptocurrencies such as Bitcoin or Ethereum." />
+                </P>
+                <Mutation
+                  mutation={editAccountSettingsMutation}
+                  refetchQueries={[{ query: collectiveSettingsV1Query, variables: { slug: collective.slug } }]}
+                  awaitRefetchQueries
+                >
+                  {(editSettings, { loading }) => (
+                    <StyledCheckbox
+                      name="crypto-contributions"
+                      label={intl.formatMessage({
+                        id: 'tier.cryptoContributions.label',
+                        defaultMessage: 'Enable crypto contributions',
+                      })}
```

Comment:
https://github.com/opencollective/opencollective-frontend/pull/8612/files#diff-27962117ecc416464040c38108900decb83aba7dbd5fd64d701018c68cf96c74

Reply:
This one I just did that because the previous title is `Enable flexible contributions`. So wanted to be consistent. But I am okay either way as it's a minor thing, so reverted it as you suggested.